src/pages/patientView/PatientViewPage.tsx

```diff
@@ -1450,6 +1451,25 @@ export default class PatientViewPage extends React.Component<
                                             />
                                         </MSKTab>
                                     )}
+                                <MSKTab
+                                    key={8}
+                                    id={PatientViewPageTabs.PATHWAY_MAPPER}
+                                    linkText={'Pathways'}
+                                >
+                                    {true /*showPM ? ( control if the values are complete*/ ? (
```

Comment:
Do we need to wait for any additional data here? If not, then we should remove the condition.